protocols/isis/server/net_ifa.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 var (
-	allISNetworkEntitiesAddr = ethernet.MACAddr{0x00, 0x2B, 0x00, 0x00, 0x05}
+	allISNetworkEntitiesAddr = ethernet.MACAddr{0x09, 0x00, 0x2B, 0x00, 0x00, 0x05}
```

Comment:
Out of curiosity: Did we only have the last 5 blocks here for a reason before? :-)

Reply:
As MAC addresses are usually 6 bytes I guess I missed a byte here when I initially wrote this.